packages/report-server/src/aggregator/Aggregator.ts

```diff
@@ -1,56 +1,38 @@
 import { Aggregator as BaseAggregator } from '@tupaia/aggregator';
-import { QueryBuilder } from './QueryBuilder';
-import { buildAggregationOptions } from './buildAggregationOptions';
+import { getDefaultPeriod } from '@tupaia/utils';
+import { Event } from '../types';
 
 export class Aggregator extends BaseAggregator {
-  constructor(dataBroker, routeHandler) {
-    super(dataBroker);
-    this.routeHandler = routeHandler;
-  }
-
   async fetchAnalytics(
-    dataElementCodes,
-    originalQuery,
-    replacementValues,
-    initialAggregationOptions = {},
+    dataElementCodes: string[],
+    organisationUnitCodes: string,
+    period = getDefaultPeriod(),
   ) {
-    const queryBuilder = new QueryBuilder(originalQuery, replacementValues, this.routeHandler);
-    // const dataSourceEntities = await queryBuilder.getDataSourceEntities();
-    const hierarchyId = await fetchHierarchyId(this.dataBroker.models, originalQuery);
-
-    const fetchOptions = await queryBuilder.build();
-
-    const entityAggregationOptions = queryBuilder.getEntityAggregationOptions();
-    const aggregationOptions = await buildAggregationOptions(
-      initialAggregationOptions,
-      [],
-      entityAggregationOptions,
-      hierarchyId,
+    return super.fetchAnalytics(
+      dataElementCodes,
+      {
+        organisationUnitCodes: organisationUnitCodes.split(','),
+        period,
+        dataServices: [{ isDataRegional: true }],
+      },
+      { aggregationType: 'RAW' },
```

Comment:
It's a pity that `aggregateAnalytics` uses `MOST_RECENT` by default (one would expect it to be `RAW`)...